scrapy/extensions/feedexport.py

```diff
@@ -195,8 +195,7 @@ def item_scraped(self, item, spider):
         return item
 
     def _load_components(self, setting_prefix):
-        conf = dict(self.settings['%s_BASE' % setting_prefix])
-        conf.update(self.settings[setting_prefix])
+        conf = dict(self.settings[setting_prefix])
```

Comment:
Shouldn't we keep '_BASE' loading an updating here for backward compatibility as the other cases?